content/applications/general/iot/devices/printer.rst

```diff
@@ -94,29 +97,52 @@ Slip`. A key can also be added to trigger the action.
 .. seealso::
    :ref:`workcenter_iot`
 
+.. _iot/link-printer:
+
 Link printer to reports
 -----------------------
 
-It's also possible to link a type of report to a certain printer. In the *IoT app*, go to the
-:guilabel:`Devices` menu, and select the desired printer that needs to be configured.
+It is possible to link type of report to a specific printer. In the *IoT app*, go to the
+:guilabel:`Devices` menu, and select the desired printer to be configured.
 
 From here, click :guilabel:`Edit`, go to the :guilabel:`Printer Reports` tab, and select
 :guilabel:`Add a line`. In the window that appears, check all the types of :guilabel:`Reports` that
 should be linked to this printer.
 
-.. image:: printer/printers-listed.png
+.. image:: printer/printer-reports.png
    :align: center
-   :alt: The printer devices listed in the IoT Devices menu.
+   :alt: The list of reports assigned to a printer in the IoT app.
+
+.. tip::
+   Reports can also be configured in the *Technical* menu while in :ref:`debug mode
+   <developer-mode>`. To do that, navigate to :menuselection:`Settings App --> Technical -->
+   Actions: Reports`. From here, the individual report can be found in this list, where the
```

Comment:
I'm having trouble finding the Actions: Reports option in the technical menu in 17.0 and 17.2. This may be because I don't have a printer to link and follow the above flow. If that's the case, it could be worth mentioning that it's necessary to link a printer before this option appears in the menu.

Reply:
Feel free to ignore this, I found the issue. The technical menu wasn't fully displaying on my portrait-mode monitor until I refreshed the page. There's no issues with the flow even without a printer linked. Sorry for the mix-up!

Reply:
No worries, thank you for being thorough!